__tests__/src/index.spec.js

```diff
@@ -111,16 +94,18 @@ describe('toMatchImageSnapshot', () => {
         added: true,
       },
     };
-    const mockDiffResult = { updated: true, code: 7 };
+    const mockDiffResult = { updated: true, pass: true };
 
     setupMock(mockDiffResult);
     const { toMatchImageSnapshot } = require('../../src/index');
     const matcherAtTest = toMatchImageSnapshot.bind(mockTestContext);
 
     const customDiffConfig = { threshold: 0.3 };
-    matcherAtTest('pretendthisisanimagebuffer', { customDiffConfig });
+    const result = matcherAtTest('pretendthisisanimagebuffer', { customDiffConfig });
     const { diffImageToSnapshot } = require('../../src/diff-snapshot');
     expect(diffImageToSnapshot.mock.calls[0][0].customDiffConfig).toBe(customDiffConfig);
+    expect(result.pass).toEqual(true);
```

Comment:
not sure I see the value in these assertions for these tests.

Reply:
result.pass was a bit unnecessary I guess. The message assertion was to pass coverage.